### DISCUSSION TITLE
A little help with my configuration

### DISCUSSION THREAD

**Discussion by Ctrl-Alb** [fastfetch-cli/fastfetch#1367] [Q&A] [answered]
Hi!
A new Fastfetch user here :)
I have a problem with my configuration, hope someone can help me.
The text in the "custom" category is formatted slightly differently than the "key" text.
What should I do to make Fastfetch display both types of text in the same way?
Thank you for your help

<img width="110" alt="Screenshot" src="https://github.com/user-attachments/assets/73bf0c17-86bb-4623-8007-7c47ae6bbcf7">

 ```
{
    "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
    "logo": {
        "padding": {
            "top": 5
        }
    },
    "display": {
        "separator": " -> "
    },
    "modules": [
       {
           "type": "title",
           "color": {
               "user": "cyan",
               "at": "cyan",
               "host": "cyan"
           }
       },
       "separator",
       {
           "type": "custom",
           "format": "{#green}모 Hardware"
       },
        {
            "type": "host",
            "key": "├ Model",
            "keyColor": "green"
        },
       {
           "type": "display",
           "key": "├ Display",
           "keyColor": "green"
       },
```

**Comment by CarterLi** [comment]
`모` uses 2 spaces, while `├` uses only one.

You should either use [nerd characters](https://www.nerdfonts.com/cheat-sheet), or remove the whitespace after `모`

**Reply by Ctrl-Alb** [reply]
Hi @CarterLi , thank you for your reply.
I wasn't able to clearly express myself, English is not my native language.
The text "Hardware" is displayed slightly differently than the text "Model", "Display", etc.
It is slightly different both for the color and for the weight (I'm attaching another image where it's easier to notice).
This happends even though I chose the same colour.
The text "Hardware" is in the "custom" category while "Model", "Display", etc are in the "key" category.
![Screenshot](https://github.com/user-attachments/assets/b80ba079-9ade-4614-8f41-3a07b403548f)

**Reply by CarterLi** [reply, marked as answer]
Logo and keys are displayed in bold by default. Use `--bright-color false` to disable it. Or you may make the text `Hardware` in bold too

```jsonc
{
    "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
    "logo": {
        "padding": {
            "top": 5
        }
    },
    "display": {
        "separator": " -> ",
        "color": {
            "keys": "green"
        }
    },
    "modules": [
        {
            "type": "title",
            "color": {
                "user": "cyan",
                "at": "cyan",
                "host": "cyan"
            }
        },
        "separator",
        {
            "type": "custom",
            "format": "{#keys}{#1}모 Hardware" // {#1} means bold / light
        },
        {
            "type": "host",
            "key": "├ Model"
        },
        {
            "type": "display",
            "key": "├ Display"
        }
    ]
}
```

**Reply by Ctrl-Alb** [reply]
Great, it worked!
Thank you very much, you've been very kind
:)
